templates_jinja2/add_data.html

```diff
@@ -0,0 +1,59 @@
+{% extends "base.html" %}
+
+{% block title %}Add Your Own Data - The Blue Alliance{% endblock %}
+
+{% block meta_description %}Instructions to add your own FRC data to The Blue Alliance{% endblock %}
+
+{% block content %}
+<div class="container">
+  <div class="row">
+    <div class="col-sm-3 col-lg-2">
+      <div class="tba-sidenav-affixed">
+        <h1 class="visible-xs end_header">Add Data to TBA!</h1>
+
+        <div class="tba-sidebar">
+          <ul class="nav tba-sidenav">
+              <li><a class="smooth-scroll" href="#match-videos">Match Video</a></li>
+              <li><a class="smooth-scroll" href="#webcasts">Event Webcasts</a></li>
+              <li><a class="smooth-scroll" href="#team-media">Team Media</a></li>
+              <li><a class="smooth-scroll" href="#offseason">Offseason Event</a></li>
+              <li><a class="smooth-scroll" href="#results">Event Results</a></li>
+              <li><a class="smooth-scroll" href="#other">Other</a></li>
+          </ul>
+        </div>
+      </div>
+    </div>
+    <div class="col-xs-12 col-sm-9 col-lg-10">
+      <h1 class="hidden-xs end_header">Add Data to TBA!</h1>
+      <p>Do you have some data you'd like to add to The Blue Alliance? This page will provide you with the resources you need to help maintain TBA.</p>
+
+      <h2 id="match-videos">Adding Match Videos</h2>
+      <p>If you have links to YouTube recording of match video, you can link them on TBA. Suggest videos individually from the detail page for a given match. Or if you have a YouTube playlist for an event, click the "Add Videos" button on the event details page and input the playlist link. Or, if you have the videos on your local computer, you can <a href="/request/apiwrite">request write keys</a> to directly add them.</p>
```

Comment:
"Or, if you have the videos on your local computer, you can request write keys to directly add them." > "Or if you're webcasting and archiving an event and want to directly add videos as they get uploaded, you can request write keys."

Or something along those lines.